x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/header_actions/header_actions.tsx

```diff
@@ -27,13 +27,7 @@ export const getHeaderActions = (
     ...(isCrawlerIndex(indexData) && indexData.connector ? [<CrawlerStatusIndicator />] : []),
     ...(isConnectorIndex(indexData) || connector ? [<SyncsContextMenu />] : []),
     ...(hasAppSearchAccess && indexData
```

Comment:
You can remove `hasAppSearchAccess` from this check (we actually should've removed that earlier).

Reply:
done